### ISSUE TITLE
[JetBrains.DataGrip] it reports an update version even if installed the latest one

### ISSUE THREAD

**Issue by ali50m** [microsoft/winget-pkgs#140633]
### Brief description of your issue

I updated JetBrains.DataGrip to version 2023.3.4  through winget. But the winget still report there is an update for JetBrains.DataGrip. Just like the issue I reported on microsoft/winget-cli#4019.

### Steps to reproduce

```
winget install JetBrains.DataGrip
```
then check update
```
winget update
```

### Expected behavior

won't see update for `JetBrains.DataGrip`.

### Actual behavior

when excute `winget update` in cli, it will promps there is an **update** for `JetBrains.DataGrip`.

### Environment

```shell
win10-64bit 21h2
winget v1.6.3482
JetBrains.DataGrip 2023.3.4
```

**Comment by mdanish-kh** [comment]
Looks like the DisplayVersion value used in the latest manifest is incorrect, I've opened a PR to fix it

**Comment by mdanish-kh** [comment]
@ali50m - This should be resolved now. Can you confirm?

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in PR #140778;
